freqtrade/rpc/webhook.py

```diff
@@ -28,12 +29,16 @@ def __init__(self, rpc: RPC, config: Dict[str, Any]) -> None:
         super().__init__(rpc, config)
 
         self._url = self._config['webhook']['url']
-
         self._format = self._config['webhook'].get('format', 'form')
+        self._retries = self._config['webhook'].get('retries', 0)
+        self._retry_delay = self._config['webhook'].get('retry_delay', 0.1)
+
+        if self._retries < 0: self._retries = 0
+        if self._retry_delay < 0: self._retry_delay = 0
```

Comment:
You can combine this with the lines above with `max(0, self._config['webhook'].get('retries', 0))` (and retry delay obviously).

potentially/most likely even better, add this as property to `constants.py` (search for webhooks) - and specify "minimum" as 0 (there's other examples for this there). 

That way you can rely config validation to give you only correct values, and freqtradebot will fail to start in case of wrong values (which is usually better than silently fixing values to something the user may not desire).

Reply:
Ok, I did this for my new params. But I noticed that 'url' and 'format' has not been defined in constants.py. Format could also potentially be defined as an enum, then presumably the validation within webhook.py would not be needed for that either. I can also see that the ...fill events are not defined in constants.py. Do you want me to clean all this up for consistency, or perhaps do it as a separate PR?

Reply:
i noticed this myself while making the above comment.

We'll for sure want to have this in the validation (and correct) - but i won't request fixing existing issues as change to a PR that's not "really" related to that - so it's up to you if you want to fix it here, separately, or leave it up to someone else (me) to fix this afterwards :)

Reply:
Done. Saved you the hassle :)